whitepaper/whitepaper.md

```diff
@@ -124,12 +133,248 @@ Blockchain technologies are growing in usage, but fragmentation is a big problem
+
+## 1.1 Terminology of Blockchain Interoperability
+
+
+This section acts as a building block to describe the different flavors of blockchain interoperability. Business use cases will be built on these simple foundation blocks leveraging a mix of them simultaneously and even expanding to several blockchains interacting concurrently. 
+
+
+### 1.1.1 Ledger Object Types
+
+
+To describe typical interoperability patterns between different blockchains, the types of objects stored on a ledger have to be distinguished. The following three types of objects stored on a ledger are differentiated as follows:
+
+* FA: Fungible asset (value token/coin)  – cannot be duplicated on different ledgers
+* NFA: Non-fungible asset – cannot be duplicated on different ledgers<sup>[d](#12-footnotes)</sup>
+* D: Data – can be duplicated on different ledgers
+
+
+Difference between a fungible (FA) and a non-fungible asset (NFA)
+
+A fungible asset is an asset that can be used interchangeably with another asset of the same type, like a currency. For example, a 1 USD bill can be swapped for any other 1 USD bill. Cryptocurrencies, such as ETH (Ether) and BTC (Bitcoin), are FAs. A non-fungible asset is an asset that cannot be swapped as it is unique and has specific properties. For example, a car is a non-fungible asset as it has unique properties, such as color and price. CryptoKitties are NFAs as well. There are two standards for fungible and non-fungible assets on the Ethereum network (ERC-20 Fungible Token Standard and ERC-721 Non-Fungible Token Standard). 
+
+Difference between an asset (FA or NFA) and data (D)
+
+Unicity applies to FAs and NFAs meaning it guarantees that only one valid representation of a given asset exists in the system. It prevents double-spending of the same token/coin in different blockchains. The same data package can have several representations on different ledgers while an asset (FA or NFA) can have only one representation active at any time, i.e., an asset exists only on one blockchain while it is locked/burned on all other blockchains. If fundamental disagreement persists in the community about the purpose or operational upgrades of a blockchain, a hard fork can split a blockchain creating two representations of the same asset to coexist. For example, Bitcoin split into Bitcoin and Bitcoin Cash in 2017. Forks are not addressing blockchain interoperability so the definition of unicity applies in a blockchain interoperability context. A data package that was once created as a copy of another data package might divert from its original one over time because different blockchains might execute different state changes on their data packages. 
```

Comment:
@DOBEN This just a question, not actual feedback. 
Would be curious to hear your thoughts about breaking the NFA category into two distinct ones one for digital NFA and one for physical (not saying we should, just wondering).
The reasoning is that the digital NFAs have their counterfeit-resistance backed by hard math/cryptography, but if I have a car tokenized on a ledger as an NFA then the only thing that cannot be counterfeited is the digital token not the car itself (I can create the same car and make sure even the serial numbers are a match).
While thinking and reading about supply chain and blockchain based solutions to it, I often noticed this pattern of misunderstanding by people where they thought that because a piece of information is signed and consensus was reached on it, then the claims contained within that piece of information must be true as well, but in reality there's only proof of someone making said claims, not the claims being true.

Reply:
The last sentence can be summarized: Garbage in => Garbage out
Garbage (false claims) that are put into the blockchain and are then signed and included as a transaction in the blockchain through the consensus algorithm will still result in garbage (false claims) coming out of the blockchain.

The weakest point at any kind of blockchain NFA solution that tracks corresponding physical assets is always the link between the physical asset identifier (Serial number, QR code, barcodes, and RFID) and the physical asset. People will always be able to counterfeit an asset and copy-paste the serial number, QR code... of the genuine asset tracked on the blockchain on to the fake asset. Unfortunately, people don’t even have to make the effort most of the time to produce/counterfeit the same asset or a similar-looking asset, because as long as an asset identifier is copied-pasted on to a fake asset, the average user will just scan the QR code or look up the serial number on the blockchain and believe whatever claims are presented. There are ways around to use solutions by companies like Scantrust and AccessReal (i-Sprint) .... Nonetheless, there is no way in the physical world to secure 100% a physical asset to its identifier but it can be made as hard as possible to break the link. As long as it takes more effort (time or money) to break the link, it is good enough for most blockchain solutions. On the one hand, the cheaper an asset the easier it is to find a solution that is good enough to link the physical asset to its identifier. On the other hand, the more expensive an asset the more likely that there is a business use case to track the asset on a blockchain.

That said, digital NFAs are always counterfeit-resistance but physical NFAs might or might not be counterfeit-resistance. I hope that people will take this problem seriously in the future because there is no added value of tracking an NFA on the blockchain that can be easily counterfeited in the physical world.